namespaces/Representor.md

```diff
@@ -0,0 +1,44 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Element)
+
+The Resource represents a Hypermedia Resource, with its available transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `attributes` (object)
+    - `transitions` - (array[Transition]) - The available transitions from this resource.
+    - `resources` - (array[Resource]) - Any embedded (resources) from this resource.
+    - `relation` - (string) - An optional relation to this resource, applicable when embedded in another resource.
+- `content` (object) - Any properties of the resource.
+
+## Transition (Object Element)
+
+A transition is an available progression from one state to another state.
+
+### Properties
+
+- `element`: transition (string, fixed)
+- `attributes` (object, required)
+    - `relation` - (string, required) - The relation of this transition from the current resource.
+    - `parameters` ([Object Type][]) - A collection of parameters for this transition.
+    - `attributes` ([Object Type][]) - A collection of possible attributes for this transition.
```

Comment:
@smizell 

> Second, I am not sure about mixing primitive JSON objects with Refract elements like this. Notice here how the attributes property is an enum of a primitive object and an array of Member Elements. This means that using attributes as on object is simply a shortcut for writing out all of these member elements you would normally use. Here, though, you have a normal object with MSON objects mixed in. This does not seem correct to me.


I actually do not like this policy "its either plain hash OR array of elements". It actually feels like it is complicating the things (re Jirka's confusion). 

If you are going to constrain it even more in a sense "if it is a plain hash the values CANNOT be elements" I would vote for dropping the "plain hash" completely. :(